content/source/docs/enterprise/system-overview/architecture.html.md

```diff
@@ -5,24 +5,201 @@ page_title: "Architecture Summary - System Overview - Terraform Enterprise"
 
 # Terraform Enterprise Architecture
 
-This page describes aspects of the architecture of Terraform Enterprise.
+Terraform Enterprise is powered by the same set of services behind HashiCorp’s public SaaS application Terraform Cloud. The sections below present the functional layers of the architecture, how they interact and details pertaining to the corresponding Docker containers involved. References to Docker containers in this document are those you will see when running a `docker ps` on a Terraform Enterprise application server while the application is up.
 
--> **Note:** This page was written for a prior implementation of Terraform Enterprise, and may be outdated. We plan to review our architecture documentation in the near future.
+~> **Advanced:** The content in this document is not required to operate Terraform Enterprise but is provided to help advanced users understand the product in more depth.
 
-## Services
+~> **Note:** Several services described below include the legacy string `ptfe` as part of their names. See [here](https://www.terraform.io/docs/enterprise/index.html#note-about-product-names) for more details.
```

Comment:
This is not technically a "legacy" string if there hasn't been an deprecation notice about replacing it. As much as I too want it gone.

Reply:
I'll pull `legacy`